src/index.js

```diff
@@ -57,6 +57,10 @@ for (const k in plugins) {
 	}
 }
 
+// @ts-ignore
+import {version} from '../package.json';
+Chart.version = version;
```

Comment:
Maybe we should put this in `core.controller` instead so that if we make another entry point for tree shaking it's still present?

Reply:
Moved, requiring the changes from couple other PR's now though.